The Manager throws an exception when creating a business

<!--
Thanks for contributing to InLong!
Please explain your issue precisely, and if possible provide a reproducer
snippet (this helps resolve issues much quicker).
-->

#### Expected behavior

When deployed by `docker-compose` and new a business, the manager should call tube-manager and create a topic.

#### Actual behavior

The manager throws an exception like below.
```
2021-08-19 10:35:15.325 [async-process-event-notifier-0] INFO  o.a.i.m.s.t.mq.CreateTubeTopicTaskEventListener:68 - finish to create tube topic for bid=b_libw_test_3
2021-08-19 10:35:15.342 [async-task-event-notifier-0] INFO  o.a.i.m.s.t.m.CreateTubeConsumeGroupTaskEventListener:69 - try to create consumer group for bid b_libw_test_3
2021-08-19 10:35:15.346 [async-task-event-notifier-0] INFO  o.a.i.m.service.core.impl.BusinessServiceImpl:127 - success to get business info
2021-08-19 10:35:15.346 [async-process-event-notifier-0] INFO  o.a.i.manager.service.core.impl.StorageServiceImpl:292 - success to filter stream
2021-08-19 10:35:15.347 [async-task-event-notifier-0] ERROR o.a.i.m.service.thirdpart.mq.TubeMqOptService:126 - fail to query tube topic libw_test_3
java.lang.IllegalArgumentException: Expected scheme-specific part at index 15: tubemq-manager:
	at java.net.URI.create(URI.java:852)
	at org.springframework.web.util.DefaultUriBuilderFactory$DefaultUriBuilder.createUri(DefaultUriBuilderFactory.java:411)
	at org.springframework.web.util.DefaultUriBuilderFactory$DefaultUriBuilder.build(DefaultUriBuilderFactory.java:404)
	at org.springframework.web.util.DefaultUriBuilderFactory.expand(DefaultUriBuilderFactory.java:154)
	at org.springframework.web.client.RestTemplate.execute(RestTemplate.java:710)
	at org.springframework.web.client.RestTemplate.exchange(RestTemplate.java:602)
	at org.apache.inlong.manager.common.util.HttpUtils.request(HttpUtils.java:57)
	at org.apache.inlong.manager.service.thirdpart.mq.TubeMqOptService.queryTopicIsExist(TubeMqOptService.java:119)
	at org.apache.inlong.manager.service.thirdpart.mq.CreateTubeConsumeGroupTaskEventListener.listen(CreateTubeConsumeGroupTaskEventListener.java:79)
	at org.apache.inlong.manager.workflow.core.event.LogableEventListener.executeListenerWithLog(LogableEventListener.java:80)
	at org.apache.inlong.manager.workflow.core.event.LogableEventListener.listen(LogableEventListener.java:58)
	at org.apache.inlong.manager.workflow.core.event.task.TaskEventNotifier.lambda$null$1(TaskEventNotifier.java:97)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.URISyntaxException: Expected scheme-specific part at index 15: tubemq-manager:
	at java.net.URI$Parser.fail(URI.java:2847)
	at java.net.URI$Parser.failExpecting(URI.java:2853)
	at java.net.URI$Parser.parse(URI.java:3056)
	at java.net.URI.<init>(URI.java:588)
	at java.net.URI.create(URI.java:850)
	... 14 common frames omitted

```

--- a/docker/docker-compose/docker-compose.yml
+++ b/docker/docker-compose/docker-compose.yml
@@ -81,7 +81,7 @@ services:
       - JDBC_URL=mysql:3306
       - USERNAME=root
       - PASSWORD=inlong
-      - TUBE_MANAGER=tubemq-manager:8089
+      - TUBE_MANAGER=http:\/\/tubemq-manager:8089
       - TUBE_MASTER=tubemq-server:8715
       - ZK_URL=tubemq-server:2181
       - SORT_APP_NAME=inlong_hive

